### DISCUSSION TITLE
Can't I use just ONE hotkey for both of 'Force light mode hotkey' and 'Force dark mode hotkey'?

### DISCUSSION THREAD

**Discussion by Kplus98** [AutoDarkMode/Windows-Auto-Night-Mode#840] [Q&A] [answered]
I want use 'Ctrl+Win+x'as the only ONE hotkey changing light mode and dark mode. But it doesn't work if i assign 'Ctrl+Win+x' to both of 'Force light mode hotkey' and 'Force dark mode hotkey'. So now I use 'Ctrl+Win+x' as 'Force light mode hotkey' and 'Ctrl+Win+z' as 'Force dark mode hotkey'. But I still want just ONE hotkey for switching light mode and dark mode.

**Comment by Armin2208** [comment, marked as answer]
You can assign Ctrl+Win+x to **Toggle theme hotkey** :)

**Reply by Kplus98** [reply]
Thank you. And thanks for making this good program :)
